### PR TITLE
Fixed the error of radio buttons, not working after saving settings.

### PR DESCRIPTION
**Pull request by vermadarsh**
This PR pertains to issue #1584 

This PR fixes the `checked` state of the radio buttons in plugin admin settings. The radio buttons after being checked were not showing as checked after saving settings.

Screenshots:
Privacy Settings: https://prnt.sc/rjj4c4
List Media View: https://prnt.sc/rjj4qc

### CODE REVIEW

File: app/admin/RTMediaFormHandler.php [hunk 1]
```diff
@@ -170,7 +170,7 @@ public static function radio( $args ) {
 		foreach ( $args['radios'] as $value => $key ) {
 			$args['rtForm_options'][] = array(
 				$key      => $value,
-				'checked' => ( (int) $args['default'] === $value ) ? true : false,
+				'checked' => ( $args['default'] == $value ) ? true : false, // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison The == error ignored as the privacy levels indexes are integer, and while saving in database, the saved value becomes string.
```
Comment (SID177) [inline app/admin/RTMediaFormHandler.php]:
```suggestion
				'checked' => ( $args['default'] == $value ) ? true : false, // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison -- The == error ignored as the privacy levels indexes are integer, and while saving in database, the saved value becomes string.
```

Did you test this by running PHPCS? It'll give error for loose comparison in your current code. `// phpcs:ignore` won't work like this. Please test before committing the code.

Comment (vermadarsh) [reply app/admin/RTMediaFormHandler.php]: Added `--` in phpcs ignore comment.

File: app/admin/RTMediaFormHandler.php [hunk 2]
```diff
@@ -170,7 +170,7 @@ public static function radio( $args ) {
 		foreach ( $args['radios'] as $value => $key ) {
 			$args['rtForm_options'][] = array(
 				$key      => $value,
-				'checked' => ( (int) $args['default'] === $value ) ? true : false,
+				'checked' => ( strval( $args['default'] ) === strval( $value ) ) ? true : false, // The function 'strval()' is used to fix the privacy levels being integer to match the saved privacy value.
```
Comment (SID177) [inline app/admin/RTMediaFormHandler.php]: Keep the comment above the code.

Comment (vermadarsh) [reply app/admin/RTMediaFormHandler.php]: I moved the comment above the code.

**Pull request merged**